tsdapiclient/tacl.py

```diff
@@ -473,7 +483,8 @@ def cli(
             if not api_key:
                 api_key = get_api_key(env, pnum)
             username, password, otp = get_user_credentials()
-            token = get_jwt_tsd_auth(env, pnum, api_key, username, password, otp, token_type)
+            token = get_jwt_two_factor_auth(env, pnum, api_key, username, password, otp, token_type, 
+                                            auth_method="iam" if env.startswith('ec-') else "tsd")
```

Comment:
Would it make sense to define the `auth_method` basic on based on active `env` for the session at a higher level. I like everything else in the PR, but having the `auth_method` deduced in two separate locations is not ideal.

Reply:
good point corrected